### PR TITLE
feat: add icons and changed banner for demo

### PR DESCRIPTION
**Pull request by waynewee**
[180613150](https://www.pivotaltracker.com/story/show/180613150)

### CODE REVIEW

File: src/components/Demo/DemoCreate/DemoCreateIssue/index.tsx [hunk 1]
```diff
@@ -75,7 +76,11 @@ export const DemoCreateIssue: FunctionComponent = () => {
                 .
               </p>
               <Link to="/demo/verify">
-                <Button className="bg-cerulean text-white rounded hover:bg-cerulean-500">Test Your File Now</Button>
+                <Banner
+                  className="text-left"
```
Comment (simboonlong) [inline src/components/Demo/DemoCreate/DemoCreateIssue/index.tsx]: think need to pass in `to`, cos currently will go to contact page, which is incorrect

File: src/components/Demo/DemoCreate/DemoCreateStart/index.tsx [hunk 2]
```diff
@@ -37,12 +37,34 @@ export const DemoCreateStart: FunctionComponent = () => {
...
+    },
+  ];
+
   return (
     <>
-      <h4 className="mt-8">Create CoO</h4>
-      <p className="mt-8">
-        See how a TradeTrust Document can be issued and provide your bank the assurance of document integrity
-      </p>
+      <p className="mt-8">Here&apos;s what you can do with this Demo:</p>
```
Comment (simboonlong) [inline src/components/Demo/DemoCreate/DemoCreateStart/index.tsx]: why `&apos;`

File: src/components/Demo/DemoCreate/DemoCreateStart/index.tsx [hunk 3]
```diff
@@ -37,12 +37,34 @@ export const DemoCreateStart: FunctionComponent = () => {
...
+      <div className="flex justify-around">
+        {features.map(({ img, title }) => {
+          return (
+            <div key={img} className="w-1/3 px-3">
+              <img className="mx-auto" src={`/static/images/demo/${img}`} />
```
Comment (simboonlong) [inline src/components/Demo/DemoCreate/DemoCreateStart/index.tsx]: add alt but up to u, i always forget also

File: src/components/UI/Banner/Banner.tsx [hunk 4]
```diff
@@ -16,13 +17,13 @@ export const Banner: FunctionComponent<BannerProps> = ({ className, title }: Ban
...
           <div className="flex flex-wrap items-center px-4 py-6 text-white">
-            <div className="px-2 w-full lg:w-1/2 mb-2 lg:mb-0">
+            <div className="px-2 w-full lg:w-2/3 mb-2 lg:mb-0">
               <h3 data-testid="banner-title">{title}</h3>
             </div>
             <div className="px-2 w-auto lg:ml-auto">
               <NavLink to="/contact">
                 <Button className="bg-tangerine text-white hover:bg-tangerine-600 border-none" size={ButtonSize.LG}>
-                  <h3>Contact us now</h3>
+                  <h4>{buttonText || "Contact us now"}</h4>
```
Comment (simboonlong) [inline src/components/UI/Banner/Banner.tsx]: lets make `buttonText` compulsory then
```suggestion
                  <h4>{buttonText}</h4>
```

**Comment by simboonlong** [comment]
the icon images looks pixelated, able to export them at @2x size for retina devices?

File: src/components/Demo/DemoCreate/DemoCreateStart/index.tsx (hunk 2 above)
Comment (waynewee) [reply src/components/Demo/DemoCreate/DemoCreateStart/index.tsx]: &apos; is ' character

Comment (simboonlong) [reply src/components/Demo/DemoCreate/DemoCreateStart/index.tsx]: yeah why needa specially use html entity? why not `'` directly

Comment (waynewee) [reply src/components/Demo/DemoCreate/DemoCreateStart/index.tsx]: error  `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`  react/no-unescaped-entities

Comment (waynewee) [reply src/components/Demo/DemoCreate/DemoCreateStart/index.tsx]: ok but i read that &#39 is the correct html escape entity

File: src/common/utils/getDropzoneBoxUi.ts [hunk 5]
```diff
@@ -2,10 +2,10 @@ enum DropzoneBoxUiState {
   DRAG_REJECT = "border-red-400 bg-red-100",
   DRAG_ACTIVE = "border-green-400 bg-green-50",
   DRAG_ACCEPT = "border-green-400 bg-green-50",
-  VERIFICATION_PENDING = "border-cloud-100 bg-white",
+  VERIFICATION_PENDING = "border-red-400 bg-white",
   VERIFICATION_ERROR = "border-red-400 bg-red-100",
   ACTION_ERROR = "border-red-400 bg-red-100",
-  DEFAULT = "border-cloud-100 bg-white",
+  DEFAULT = "border-red-400 bg-white",
```
Comment (simboonlong) [inline src/common/utils/getDropzoneBoxUi.ts]: last thing, why became border-red for the neutral states?

Comment (waynewee) [reply src/common/utils/getDropzoneBoxUi.ts]: its in the [figma](https://www.figma.com/file/8GOvq1FnnTvxuKQydN9tBJ/TradeTrust---redesign-R2?node-id=2797%3A111), think gta ask jerry

**Review by simboonlong — APPROVED** [review]

**Pull request merged**